priv/scripts/otp/otp.sh

```diff
@@ -5,12 +5,23 @@ set -euox pipefail
 ref_name=$1
 ref=$2
 linux=$3
+arch=$4
 
 source ${SCRIPT_DIR}/utils.sh
 
-echo "Building $1 $2 $3"
+echo "Building $1 $2 $3 $4"
+
+if [[ "$arch" == "arm64" ]]; then
+  container="otp-build-${linux}-${ref_name}-arm"
+  platform="--platform linux/arm64"
```

Comment:
This flag and buildx is not necessary. We will run the builds on machines with different architectures so the --platform is not necessary.